infrastructure/variables.tf

```diff
@@ -17,4 +17,9 @@ variable "env" {
   type = "string"
 }
 
+variable "s2s_url" {
```

Comment:
I don't see what benefit this gives

Reply:
Removing 38c83df1cd545c8c70c63cfbe787182fcdc1b60f